### ISSUE TITLE
Report RTCs to local groups and/or BT

### ISSUE THREAD

**Issue by crsdc** [crsdc/badgerapp#21]
Out of scope for minimum viable v1 of the app, so marked wontfix for the foreseeable future.

Opened issue to pre-empt raising of this during testing, and to gather any discussion of it as a likely feature request.

The app will allow users to save locations locally, including RTC/RTA locations, potentially making it easier for them to note badgers killed on the roads and pass the relevant info on themselves.

**Comment by DawnBadger** [comment]
Just confirming on review of issues that BT always need the report coming to BT. If it could also go to group at same time that is a bonus, and would save triage at office end, but think would need coding and backend database of groups and postcodes which would be complex. It can't 'just' go to a local group as BT collates national stats.

**Comment by crsdc** [comment]
My comment is academic for now since no plans to implement this any time soon, but in theory we could maintain the data needed to route reports to the appropriate local group within the app, and roll out any changes with version updates. It would consist of a list of coordinates for polygons, mirroring county borders or other patches of ground, which can be lifted from public sources, plus a list of email addresses supplied by local groups.

This could all live in the app rather than a backend. So not technically dependent on #18 ... though that would make things easier if we ever did want to implement reporting to local groups.
